internal/cluster/clustersetup/activity_install_ingress_controller.go

```diff
@@ -145,7 +162,19 @@ func (i *IngressControllerPostHook) Do(cluster CommonCluster) error {
 
 	namespace := global.Config.Cluster.Namespace
 
-	return i.helmService.ApplyDeployment(context.Background(), cluster.GetID(), namespace, config.Ingress.Chart, "ingress", valuesBytes, config.Ingress.Version)
+	err = a.helmService.ApplyDeployment(
+		context.Background(),
+		input.ClusterID,
+		namespace, config.Ingress.Chart,
+		"ingress",
+		valuesBytes,
+		config.Ingress.Version)
```

Comment:
**Recommended**: it is more common/idiomatic to add a comma and break a line at the end of the last argument, because it improves readability.